readme.MD

```diff
@@ -58,6 +58,10 @@ Class `FirebaseVersionFetcher` takes optional argument for cache expiration of f
 ```groovy
 implementation "cz.ackee.updatechecker:firebase-fetcher:x.x.x"
 ```
+Besides that, you need to add a dependency to the Firebase Remote Config since it is not transitive.
+```groovy
+implementation "com.google.firebase:firebase-config:x.x.x"
```

Comment:
i would prompt to add ktx version

Reply:
780130c5415268bd18d1ef3592c92b947c3716cc